torch/fx/experimental/proxy_tensor.py

```diff
@@ -59,8 +60,12 @@ def wrap_with_proxy(e, proxy):
     else:
         return real_out
 
+from torch.testing._internal.composite_compliance import check_metadata_consistency
 
 def proxy_call(func_overload, args, kwargs=None):
+    if func_overload == torch.ops.prim.device.default:
```

Comment:
I guess kwarg cannot actually be None here otherwise things would break in the function below.

Reply:
this isn't exposed to the user, so kwargs is always None

Reply:
But you do `**kwargs` below. That is not allowed is `kwargs` is `None`.

Reply:
it will always be a dict here because this isn't user-exposed and is invoked only when the device is virtualized and device is called.
https://github.com/pytorch/pytorch/blob/591c4886ffbac6c76e17f48a241495fa2d6fdfe7/torch/csrc/autograd/python_variable.cpp#L1881 but it is maybe more defensive to add a `kwargs = kwargs if kwargs else {}`